code/modules/mob/living/carbon/human/human_emote.dm

```diff
@@ -648,6 +648,35 @@
 	muzzled_noises = list("annoyed")
 	emote_type = EMOTE_SOUND | EMOTE_MOUTH
 
+/datum/emote/living/carbon/human/hiss/tajaran
+	key = "hiss"
```

Comment:
Since the parent (`/datum/emote/living/carbon/human/hiss`) has some of these variables defined with the same exact values, you can remove them from here:
- key (pretty sure)
- key_third_person
- message
- message_param
- emote_type